gradle/scriptsTest.gradle

```diff
@@ -28,7 +28,8 @@ configurations {
 task scriptsTest(type: Test) {
     group = 'Verification'
     description = 'Runs all scripts tests'
-    testClassesDir = sourceSets.scriptsTest.output.classesDir
+
+    testClassesDirs = sourceSets.scriptsTest.output.classesDirs
```

Comment:
note - it doesn't mean we don't generate javadoc for java classes, the scaladoc will just doesn't look in the empty java output directory since we are using scala output directoru